jwst/pipeline/calwebb_image3.py

```diff
@@ -29,89 +29,63 @@ class Image3Pipeline(Pipeline):
     """
 
     spec = """
-        suffix = string(default='i2d')
     """
 
     # Define alias to steps
-    step_defs = {'assign_mtwcs': assign_mtwcs_step.AssignMTWcsStep,
-                 'tweakreg': tweakreg_step.TweakRegStep,
-                 'skymatch': skymatch_step.SkyMatchStep,
-                 'outlier_detection': outlier_detection_step.OutlierDetectionStep,
-                 'resample': resample_step.ResampleStep,
-                 'source_catalog': source_catalog_step.SourceCatalogStep
-                 }
-
-    def process(self, input):
+    step_defs = {
+        'assign_mtwcs': assign_mtwcs_step.AssignMTWcsStep,
+        'tweakreg': tweakreg_step.TweakRegStep,
+        'skymatch': skymatch_step.SkyMatchStep,
+        'outlier_detection': outlier_detection_step.OutlierDetectionStep,
+        'resample': resample_step.ResampleStep,
+        'source_catalog': source_catalog_step.SourceCatalogStep
+    }
+
```

Comment:
Just for my own edification, what's gained or changed by moving `step_defs` out of the `process` method?

Reply:
I'm not sure what you mean. `step_defs` is a class attribute, and was never part of the `process` method. I think this is actually a part of the `Pipeline` infrastructure. 

As I was going through, I just changed some indentation is all

Reply:
Ah, OK. Perhaps it was just the layout of the diff that made me think it used to be inside of `process`, but it really wasn't.